keps/sig-storage/962-execution-hook/kep.yaml

```diff
@@ -27,3 +27,6 @@ replaces:
   - n/a
 superseded-by:
   - n/a
+
+latest-milestone: "0.0"
+stage: "alpha"
```

Comment:
Sorry - I hate myself for this comment, but I'm wondering if we should consider "pre-alpha" (or sth like that) for those cases.

When I see a stage in those files, I conceptually think that this stage is happening, and in some cases it's not necessary true. (It would also be a cleaner marker).

I'm happy to be outvoted though - @ehashman @johnbelamaric @deads2k for thoughts

Reply:
I tried setting "unknown" previously but it got rejected by validation and I wasn't sure if it made sense to add it as a new state.